packages/cloud-functions/src/cico/Moonpay.ts

```diff
@@ -0,0 +1,144 @@
+import { DigitalAsset, FETCH_TIMEOUT_DURATION, MOONPAY_DATA } from '../config'
+import { PaymentMethod, ProviderQuote, UserLocationData } from './fetchProviders'
+import { bankingSystemToCountry } from './providerAvailability'
+import { fetchLocalCurrencyAndExchangeRate, fetchWithTimeout } from './utils'
+
+interface MoonpayQuote {
```

Comment:
Does this have tests?  Worth adding some simple ones if they don't exist?  I initially looked for them to try to understand expected behavior better.

Reply:
Good point I'll add some in. They won't be the most effective because documentation of the different return values of these endpoints is not comprehensive but the tests can at least be instructive of the behaviors we expect